### PR TITLE
fix build error in vercel deploy

### PR DESCRIPTION
**Pull request by huisq**
Issue Overview
as titled, error: yarn next build failed

Solution
resolved by modifying ./packages/nextjs/utils/scaffold-stark/contractsData.ts
add "decodeBigIntArrayToText"

### CODE REVIEW

File: packages/nextjs/utils/scaffold-stark/contractsData.ts [hunk 1]
```diff
@@ -5,3 +5,27 @@ export function getAllContracts() {
...
+  }
+  try {
+    return new TextDecoder("utf-8").decode(new Uint8Array(bytes));
+  } catch (e) {
+    console.error("Failed to decode:", e);
+    return "";
+  }
+}
+
+export function decodeBigIntArrayToText(byteArray: string[]) {
```
Comment (0xquantum3labs) [inline packages/nextjs/utils/scaffold-stark/contractsData.ts:26]: i think we can just remove the `import decode ...` from the `MyHoldings.tsx` file.

Comment (0xquantum3labs) [reply packages/nextjs/utils/scaffold-stark/contractsData.ts:26]: we dont need to define this function.

**Review by 0xquantum3labs — CHANGES_REQUESTED** [review]

File: packages/nextjs/utils/scaffold-stark/contractsData.ts (hunk 1 above)
Comment (0xquantum3labs) [reply packages/nextjs/utils/scaffold-stark/contractsData.ts:26]: @huisq can you update this PR please ? thanks 🙌

**Comment by huisq** [comment]
fixed PR

**Review by jrcarlos2000 — APPROVED** [review]

**Comment by jrcarlos2000** [comment]
> fixed PR

Thanks 🙌🏽

**Pull request merged**